elements/pfe-markdown/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@patternfly/pfe-markdown",
+  "pfelement": {
+    "className": "PfeMarkdown",
+    "elementName": "pfe-markdown"
+  },
+  "version": "1.0.0-prerelease.9",
+  "description": "Markdown element for PatternFly Elements",
+  "repository": {
+    "type": "git",
+    "url": ""
```

Comment:
@kylebuch8 Can you include the url here so that it shows up in npm.  We also need the directory so that it links directly to the right component.